packages/graph-editor/src/components/panels/dropPanel/dropPanel.tsx

```diff
@@ -1,12 +1,13 @@
 import { Accordion, Box, Stack, Text, TextInput } from '@tokens-studio/ui';
 import { DragItem } from './DragItem.js';
 import { DropPanelStore } from './data.js';
-import { IconoirProvider, NavArrowRight } from 'iconoir-react';
+import { IconoirProvider } from 'iconoir-react';
```

Comment:
Check where this is used, we just need a wrapper around the icons used to make sure they are the correct size